recipes/sqlean.py/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "sqlean.py" %}
+{% set version = "3.45.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sqlean.py-{{ version }}.tar.gz
+  sha256: 25fbf0a111856e0384dbc412156e2e688948525c7d8609dc3803512c70130740
+
+build:
+  number: 0
+  script:
+    - {{ PYTHON }} setup.py build_ext -i
```

Comment:
The pip command below should build the extension for you. Can you try removing this step and checking if that works?

Reply:
I tried that locally (on macos) but it did not succeed.

<img width="452" alt="Screenshot 2024-04-23 at 8 19 57 AM" src="https://github.com/conda-forge/staged-recipes/assets/80478925/5104896b-3312-4478-b187-11cec8350117">

Reply:
OK, it's passing! And I realized my mistake: I ran it with `noarch: python`.

Thanks for the review!